source/includes/stackpath/_instances.md

```diff
@@ -0,0 +1,117 @@
+### Instances
```

Comment:
I might be wrong, but I think there's an extra level of nesting here. If true, this should be `## Instances` and the operations should be `### List instances` for example

Reply:
yep. you are right. I base myself on the workload. I will change that.

Reply:
b0201713923a94550fc763500a5a5a0a264e51b3